pom.xml

```diff
@@ -124,4 +165,83 @@
     </plugins>
   </reporting>
 
+  <profiles>
+    <profile> 
+      <id>release</id>
```

Comment:
I am using the suggestion here:

http://central.sonatype.org/pages/apache-maven.html#using-a-profile

And I think the reasoning is sound.